docs/architecture/deep-dives/autofill/shadow-dom.md

```diff
@@ -45,7 +45,8 @@ If the node is a shadow root, we try to recursively descend into it using the br
 - **Firefox: **
   [Element.openOrClosedShadowRoot](https://developer.mozilla.org/en-US/docs/Web/API/Element/openOrClosedShadowRoot) -
   ignores the `mode` property and works for open and closed shadow roots
-- **Safari and others: **
+- **Safari and others:**
+
```

Comment:
Was this likebreak intentional? The other list items does not have a linebreak here.

Reply:
Fixed